### DISCUSSION TITLE
Certificate photo is not sent

### DISCUSSION THREAD

**Discussion by ravilovrm** [community/community#133086] [GitHub Education]
### Hi everyone,

Hi everyone,
I want to send a photo of a student card to receive Github Education benefits, I upload the photo from the directory on my computer, select a category and click the “Submit” button, after which the same page loads again, the email does not receive the message, therefore nothing is sent.

How to send? How to fix this bug?

**Comment by davevad93** [comment]
Hi @TUTBOROMA ,

This is an issue I have noticed that has no general solution. Try the workaround in discussion #126276, if it doesn't work then try another browser such as `Brave`. These are the only solutions I have seen that have worked for some users experiencing the same problem.
